Header header
OutputPathsMsg output_paths